ufo/static/versionDisplay.html

```diff
@@ -40,9 +58,40 @@
           value: false,
           notify: true,
         },
+        shouldShowUpdate: {
+          type: Boolean,
+          value: false,
+          notify: true,
+        },
       },
       attached: function() {
         this.versionText = __i18n__['versionText'];
+        this.versionUpdateText = __i18n__['versionUpdateText'];
+      },
+      handleResponse: function(response) {
+        this.set('shouldShowUpdate', this._isGithubVersionNewer());
+      },
+      _isGithubVersionNewer: function() {
+        if (!this.currentVersionObj || !this.currentVersionObj.version || !this.githubVersionObj || !this.githubVersionObj.version) {
+          return false;
+        }
+        var currentVersionArray = this._makeVersionStringIntoNumberArray(this.currentVersionObj.version);
+        var githubVersionArray = this._makeVersionStringIntoNumberArray(this.githubVersionObj.version);
+        var minLength = Math.min(currentVersionArray.length, githubVersionArray.length);
+        for (var i = 0; i < minLength; i++) {
+          if (currentVersionArray[i] < githubVersionArray[i]) {
+            return true;
+          }
+        }
+        return githubVersionArray.length > currentVersionArray.length;
+      },
+      _makeVersionStringIntoNumberArray: function(version) {
+        var parts = version.split('.');
+        var numbs = new Array(parts.length);
```

Comment:
nit: instead of the abbreviation, can we use "numbers"?  So much nicer to read.  :)